commands/find.go

```diff
@@ -33,6 +69,9 @@ func getCidCmd(c *cli.Context) error {
 		return err
 	}
 
-	return cl.Get(ctx, ccid)
+	resp, err := cl.Get(ctx, ccid, end)
+	log.Info("Response:")
+	resp.PrettyPrint()
```

Comment:
Pretty-printed output should be an option, not the default IMO.  If this is JSON, then people can just run it through `jq` anyway to make it pretty.